README.md

```diff
@@ -87,6 +87,7 @@ module.exports = {
 | searchResultContextMaxLength     | number                                   | `50`      | Set the max length of characters of each search result to show.                                                                                                                                                                                                                                                                                                |
 | explicitSearchResultPath         | boolean                                  | `false`   | Whether an explicit path to a heading should be presented on a suggestion template.                                                                                                                                                                                                                                                                            |
 | ignoreFiles                      | string \| RegExp \| (string \| RegExp)[] | `[]`      | Set the match rules to ignore some routes. Put a string if you want an exact match, or put a regex if you want a partial match. Note: without the website base url.                                                                                                                                                                                            |
+| ignoreClasses                    | string \| string[]                       | `[]`      | A list of html classes to ignore when indexing each page.                                                                                                                                                                                                                                                                                                      |
```

Comment:
Thanks for your contributions! How about change this option to `ignoreSelectors`? So that users can ignore not only by classes, but also by other css selectors?

Reply:
Good idea, not sure why I didn't do that from the start :) Fixed it!